### PR TITLE
updated stuff

### PR DESCRIPTION
**Pull request by TacoTheDank**
yeah

still cant fix travis tho ugh

### CODE REVIEW

File: app/build.gradle [hunk 1]
```diff
@@ -1,5 +1,5 @@
 plugins {
-    id "io.gitlab.arturbosch.detekt" version "1.0.0.RC6-3"
+    id "io.gitlab.arturbosch.detekt" version "1.0.0.RC8"
```
Comment (tasomaniac) [inline app/build.gradle]: Unfortunately this conflicts with Novoda static analysis plugin. Needs to be reverted.

File: app/build.gradle [hunk 2]
```diff
@@ -23,9 +23,9 @@ def gitSha = 'git rev-parse --short HEAD'.execute([], project.rootDir).text.trim
 
 android {
     compileSdkVersion versions.compileSdk
-
+    buildToolsVersion versions.buildTools
```
Comment (tasomaniac) [inline app/build.gradle]: This is not required. Can you remove it?

File: app/build.gradle [hunk 3]
```diff
@@ -118,11 +118,11 @@ if (isPlay) {
...
 dependencies {
-    playImplementation 'com.crashlytics.sdk.android:crashlytics:2.9.2'
-    playImplementation "com.google.firebase:firebase-core:15.0.2"
+    playImplementation 'com.crashlytics.sdk.android:crashlytics:2.9.4'
+    playImplementation "com.google.firebase:firebase-core:16.0.1"
 
-    implementation 'com.google.code.findbugs:jsr305:3.0.2'
-    implementation 'androidx.core:core-ktx:0.3'
+    implementation 'com.github.spotbugs:spotbugs:3.1.6'
+    implementation 'androidx.core:core-ktx:1.0.0-beta01'
```
Comment (tasomaniac) [inline app/build.gradle]: I don't think 1.+ Version of ktx works together with other support libs. The project needs to be jettified

File: app/build.gradle [hunk 4]
```diff
@@ -118,11 +118,11 @@ if (isPlay) {
...
 
 dependencies {
-    playImplementation 'com.crashlytics.sdk.android:crashlytics:2.9.2'
-    playImplementation "com.google.firebase:firebase-core:15.0.2"
+    playImplementation 'com.crashlytics.sdk.android:crashlytics:2.9.4'
+    playImplementation "com.google.firebase:firebase-core:16.0.1"
 
-    implementation 'com.google.code.findbugs:jsr305:3.0.2'
-    implementation 'androidx.core:core-ktx:0.3'
+    implementation 'com.github.spotbugs:spotbugs:3.1.6'
```
Comment (tasomaniac) [inline app/build.gradle]: What is Spotbugs? Can you revert this line?

File: .travis.yml [hunk 5]
```diff
@@ -6,10 +6,27 @@ language: android
 jdk:
   - openjdk8
 
+android:
```
Comment (tasomaniac) [inline .travis.yml]: Without all of this, Travis setup has been working fine. The issue on master is the library versions are outdated. There is a lint check for that. I will disable that lint check. 

I'm trying to keep the Travis setup as simple as possible.

File: app/build.gradle (hunk 1 above)
Comment (tasomaniac) [reply app/build.gradle]: Travis fails because of this line.

File: app/build.gradle (hunk 4 above)
Comment (tasomaniac) [reply app/build.gradle]: Spotbugs seems like a separate tool. I don't really want to add that as a dependency. 2 line above, it says `findbugs` in the group but the artifact is really just the `jsr305` annotations, nothing more.

File: app/build.gradle (hunk 1 above)
Comment (TacoTheDank) [reply app/build.gradle]: hmmk

File: .travis.yml (hunk 5 above)
Comment (TacoTheDank) [reply .travis.yml]: so keep them or revert it

File: app/build.gradle [hunk 6]
```diff
@@ -118,10 +117,10 @@ if (isPlay) {
 }
 
 dependencies {
-    playImplementation 'com.crashlytics.sdk.android:crashlytics:2.9.2'
-    playImplementation "com.google.firebase:firebase-core:15.0.2"
+    playImplementation 'com.crashlytics.sdk.android:crashlytics:2.9.4'
+    playImplementation "com.google.firebase:firebase-core:16.0.1"
 
-    implementation 'com.google.code.findbugs:jsr305:3.0.2'
+    implementation 'com.github.spotbugs:spotbugs:3.1.6'
```
Comment (tasomaniac) [inline app/build.gradle]: Can you also revert this line? Thanks.

Comment (TacoTheDank) [reply app/build.gradle]: kk

**Comment by TacoTheDank** [comment]
woo

**Pull request merged**